patches/server/0010-Adventure.patch

```diff
@@ -3347,6 +3354,7 @@ index 43f6ac44525048faf0b0cd2c11f9bc36fa0cab5d..1317cdd48c1c27c9e5152da5403237b1
 +
 +    @Override
 +    public void line(final int index, final net.kyori.adventure.text.@NotNull Component line) throws IndexOutOfBoundsException {
++        Preconditions.checkNotNull(line, "Line cannot be null");
```

Comment:
use `com.google.common.base.Preconditions.checkNotNull(line, "Line cannot be null");`